app/src/main/java/nl/nfi/cellscanner/Database.java

```diff
@@ -206,6 +206,11 @@ public void storeLocationInfo(Location location) {
         db.insert("locationinfo", null, values);
     }
 
+    /** drop data until a given timestamp **/
+    public void dropDataUntil(long timestamp) {
```

Comment:
```suggestion
    public void dropUploadedData(long timestamp) {
```
Misschien een duidelijkere naam

Reply:
Hij dropt data tot een bepaalde timestamp, dat dat ook de Uploaded time is is puur toeval =D